test/e2e/common_test.go

```diff
@@ -138,10 +139,29 @@ const (
 	imageCacheDir = "imagecachedir"
 )
 
+var netnsFiles []fs.DirEntry
+
+func getNetnsDir() string {
+	if isRootless() {
+		if env, ok := os.LookupEnv("XDG_RUNTIME_DIR"); ok {
```

Comment:
This relies on XDG being set in the environment that calls ginkgo. Is that a safe assumption in CI? Podman goes to some lengths to compute a default if XDG is unset.

Duplication is evil, but I think it might be worth changing this to default-if-unset like system tests do.

Reply:
I mean we can certainly do it, CI use ssh to switch to the user which should set all the proper env vars we want.